shell/components/form/Taints.vue

```diff
@@ -56,7 +56,7 @@ export default {
 <template>
   <div class="taints">
     <KeyValue
-      v-model="localValue"
+      :value="value"
```

Comment:
Funny to see that the `localValue` pattern was implemented 4 years ago..

https://github.com/rak-phillip/dashboard/commit/7a3c09dcc419b4709b41069e0e03df6e78cc9c30#diff-5b8f84a7e9a14b05f4460a3ac34988e86bde13325badca0b7eb4eaf6bb72e2d2R30-R37

I think that the computed prop becomes dead with this change, we might want to consider removing it.